resources/views/stats.blade.php

```diff
@@ -0,0 +1,64 @@
+@extends('layouts.app')
+
+@section('content')
+    <dl class="mt-5 grid grid-cols-1 gap-5 sm:grid-cols-4">
+        <div class="px-4 py-5 bg-white shadow rounded-lg overflow-hidden sm:p-6">
+            <dt class="text-sm font-medium text-gray-500 truncate">Last Week</dt>
+            <dd class="mt-1 text-3xl tracking-tight font-semibold text-gray-900">
+                <h3>{{ $week['current'] }} hits</h3>
+                <p class="text-gray-500 text-sm">Previous: {{ $week['previous'] }}</p>
+                <p class="text-gray-500 text-sm">Percent Change: {{ $week['change'] }}%</p>
+            </dd>
+        </div>
+
+        <div class="px-4 py-5 bg-white shadow rounded-lg overflow-hidden sm:p-6">
+            <dt class="text-sm font-medium text-gray-500 truncate">Last Month</dt>
+            <dd class="mt-1 text-3xl tracking-tight font-semibold text-gray-900">
+                <h3>{{ $month['current'] }} hits</h3>
+                <p class="text-gray-500 text-sm">Previous: {{ $month['previous'] }}</p>
+                <p class="text-gray-500 text-sm">Percent Change: {{ $month['change'] }}%</p>
+            </dd>
+        </div>
+
+        <div class="px-4 py-5 bg-white shadow rounded-lg overflow-hidden sm:p-6">
+            <dt class="text-sm font-medium text-gray-500 truncate">All Time</dt>
+            <dd class="mt-1 text-3xl tracking-tight font-semibold text-gray-900">
+                <h3>{{ $year['current'] }} hits</h3>
+                <p class="text-gray-500 text-sm">Previous: {{ $year['previous'] }}</p>
+                <p class="text-gray-500 text-sm">Percent Change: {{ $year['change'] }}%</p>
+            </dd>
+        </div>
+
+        <div class="px-4 py-5 bg-white shadow rounded-lg overflow-hidden sm:p-6">
+            <dt class="text-sm font-medium text-gray-500 truncate">Top Endpoints</dt>
```

Comment:
Maybe we write (all time) after this so it's clear what time period we're talking about?